parsers/BR.py

```diff
@@ -116,8 +117,8 @@ def fetch_exchange(country_code1='BR', country_code2='UY', session=None):
     gd = get_data()
 
     data = {
-        'datetime': gd['Data'],
-        'sortedCountryCodes': 'BR->UY',
+        'datetime': parser.parse(gd['Data']),
+        'sortedCountryCodes': 'UY->BR',
```

Comment:
The country codes are sorted (to disambiguate commutating pairs).
You should return BR->UY, but you can multiply the flow by -1 if needed.